pkg/engine/jmespath/functions_test.go

```diff
@@ -1675,3 +1675,61 @@ func Test_ImageNormalize(t *testing.T) {
 		})
 	}
 }
+func Test_IsExternalURL(t *testing.T) {
+	testCases := []struct {
+		jmesPath       string
+		expectedResult bool
+	}{
+		{
+			jmesPath:       "is_external_url('http://localhost')",
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://127.0.0.1')",
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://172.16.1.1')",
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://10.1.1.1')",
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://192.168.0.1')",
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://google.com')",
+			expectedResult: true,
+		}, {
+			jmesPath:       "is_external_url('http://8.8.8.8')",
+			expectedResult: true,
+		}, {
+			jmesPath:       "is_external_url('http://245.48.83.160')",
+			expectedResult: true,
+		}, {
+			jmesPath:       "is_external_url('http://0xc0.0xa8.774')", //192.168.3.6
+			expectedResult: false,
+		}, {
+			jmesPath:       "is_external_url('http://182.4048903')", //182.61.200.7
```

Comment:
Test are failing because this IP is not reachable on the github CI runner it seems.

Reply:
Thank you for pointing that out. I will update the test cases to address the issue.